aws/client.go

```diff
@@ -0,0 +1,89 @@
+package aws
+
```

Comment:
what is this file for ? Is it for accessing the s3 bucket ?

Reply:
yes, it's a wrapper for all the AWS service clients. Currently we only have S3 client.